docs/articles/documentation/continuous-integration/bitbucket-pipelines.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: docs
+title: Run Tests in Bitbucket Pipelines
+permalink: /documentation/continuous-integration/bitbucket-pipelines.html
+---
+# Run Tests in Bitbucket Pipelines CI
+
+You can automatically run TestCafe tests as a part of your build process on [Bitbucket Pipelines CI](https://bitbucket.org/product/features/pipelines).
+
+Suppose you have a Bitbucket project for which you need to automatically run tests in the cloud when the project is modified. To do this, go through the following steps:
```

Comment:
"Suppose you have a Bitbucket project for which you need to automatically run tests in the cloud when the project is modified. To do this, go through the following steps:" -> You can set up your Bitbucket project to automatically run tests in the cloud when the project is modified: